pkg/server/router.go

```diff
@@ -75,27 +73,27 @@ func (r *Router) Routes(h router.Handlers) http.Handler {
 	m.Path("/v1/harvester/{type}/{namespace}/{name}").Handler(h.K8sResource)
 	m.Path("/v1/harvester/{type}/{namespace}/{name}/{link}").Handler(h.K8sResource)
 
-	loginHandler := auth.NewLoginHandler()
-	m.Path("/v1-public/auth").Handler(loginHandler)
-
-	vueUI := ui.Vue
-	m.Handle("/dashboard/", vueUI.IndexFile())
-	m.PathPrefix("/dashboard/").Handler(vueUI.IndexFileOnNotFound())
-	m.PathPrefix("/api-ui").Handler(vueUI.ServeAsset())
-
-	if r.options.RancherEmbedded || r.options.RancherURL != "" {
-		host, scheme, err := parseRancherServerURL(r.options.RancherURL)
-		if err != nil {
-			logrus.Fatal(err)
-		}
-		rancherHandler := &proxy.Handler{
-			Host:   host,
-			Scheme: scheme,
+	// enable local dashboard ui for dev
+	if r.options.AddLocalUI {
+		vueUI := ui.Vue
+		m.Handle("/dashboard/", vueUI.IndexFile())
+		m.PathPrefix("/dashboard/").Handler(vueUI.IndexFileOnNotFound())
+		m.PathPrefix("/api-ui").Handler(vueUI.ServeAsset())
+
+		if r.options.AddLocalUI || r.options.RancherURL != "" {
```

Comment:
Everything is good except the first condition is not needed.

Reply:
fixed, thanks